lib/paperclip.rb

```diff
@@ -97,7 +97,8 @@ def self.options
       :log_command => true,
       :swallow_stderr => true,
       :content_type_mappings => {},
-      :use_exif_orientation => true
+      :use_exif_orientation => true,
+      :read_timeout => 120
```

Comment:
> Where do we get the 120 default value from?

You're right, I changed it to nil.

> Put a comma after the last item of a multiline hash.

Actually, trailing commas are not preferred: https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas.